vuu-ui/packages/vuu-theme/css/characteristics/focused.css

```diff
@@ -1,21 +1,7 @@
 .vuu-theme {
-  --salt-focused-outlineColor: var(--salt-palette-interact-outline);
-  --salt-focused-outlineStyle: dotted;
-  --salt-focused-outlineWidth: 2px;
-  --salt-focused-outlineInset: 0;
-  --salt-focused-outlineOffset: 0;
-
-  --salt-focused-outline: var(--salt-focused-outlineWidth) var(--salt-focused-outlineStyle) var(--salt-focused-outlineColor); /* CSS shortcut token: not in Figma */
+  --salt-editable-borderColor-active: var(--editable-border-active, #6D18BD);
 }
 
-.saltFocusVisible:after,
-.focused:focus:after,
-.focused:focus-visible:after {
-  content: "";
-  inset: var(--salt-focused-outlineInset);
-  outline-color: var(--salt-focused-outlineColor);
-  outline-offset: var(--salt-focused-outlineOffset);
-  outline-style: var(--salt-focused-outlineStyle);
-  outline-width: var(--salt-focused-outlineWidth);
-  position: absolute;
-}
+.saltInput-focused{
+  border-color: var(  --salt-editable-borderColor-active)!important;
+}
```

Comment:
Have you had a good check through the Showcase to make sure all other components are behaving as expected?

Reply:
I went through the app and the styling looks fine to me. I got rid of the outline styling for focused input here.  I wouldn't worry too much about this change as, looking at figma, we don't ever apply dotted outline to focused elements